### PR TITLE
TwistController Updates

### PR DESCRIPTION
**Pull request by fmessmer**
PR includes changes related to:
- new Acceleration Limiters
- Restructuring for KinematicExtensions in order to consider them in Limiters
- Limiters are also applied to KinematicExtensions
- new COB_TORSO extension
- new LOOKAT extension
- various configuration adjustments according to Hardware/Simulation tests (i.e. ipa-fxm-mb's final demo, twist_control video shoot,...)

### CODE REVIEW

**Comment by benmaidel** [comment]
FR ok

File: cob_twist_controller/CMakeLists.txt [hunk 1]
```diff
@@ -70,6 +70,17 @@ add_executable(debug_evaluate_jointstates_node src/debug/debug_evaluate_jointsta
...
+
+add_executable(test_moving_average_node src/debug/test_moving_average_node.cpp)
+add_dependencies(test_moving_average_node ${catkin_EXPORTED_TARGETS})
+target_link_libraries(test_moving_average_node ${catkin_LIBRARIES})
+
+add_executable(test_simpson_integrator_node src/debug/test_simpson_integrator_node.cpp)
+add_dependencies(test_simpson_integrator_node ${catkin_EXPORTED_TARGETS})
+target_link_libraries(test_simpson_integrator_node ${catkin_LIBRARIES})
+
+roslint_cpp()
```
Comment (NadiaHG) [inline cob_twist_controller/CMakeLists.txt:82]: Does it work?

**Comment by NadiaHG** [comment]
FR: OK

**Comment by fmessmer** [comment]
Tested on cob4-1 aka. cob4-2 without arms

File: cob_twist_controller/CMakeLists.txt (hunk 1 above)
Comment (floweisshardt) [inline cob_twist_controller/CMakeLists.txt:82]: yes, it does: see http://wiki.ros.org/roslint (but you have to call `catkin_make roslint_my_fancy_package`)

**Pull request merged**